pkg/features/kube_features.go

```diff
@@ -936,6 +936,7 @@ const (
 
 func init() {
 	runtime.Must(utilfeature.DefaultMutableFeatureGate.Add(defaultKubernetesFeatureGates))
+	runtime.Must(utilfeature.DefaultMutableFeatureGate.AddVersioned(defaultVersionedKubernetesFeatureGates))
```

Comment:
huh, it looks like `Add` was updated to try to make this unncessary.

Reply:
> huh, it looks like `Add` was updated to try to make this unncessary.

I see, the change to `.Add` don't actually bring the past up to date.  We're in a state of in-between with no featuregates explaining their history yet.